releasenotes/notes/adjust-symbolic-pulses-amp-angle-representation-f5c40007416cf938.yaml

```
@@ -0,0 +1,25 @@
+---
+other:
+  - |
+    Qiskit Terra 0.23.0, began phasing out support of complex parameters
+    in the Pulse module. Mainly, all library symbolic pulses were converted
+    from complex amplitude representation to a duo of real (float) parameters
+    (amp, angle). To avoid problems, qiskit experiments adopted this convention.
+    
+    The ``FixedFrequencyTransmon`` library now includes an ``angle`` parameter
```

```suggestion
    The :class:`.FixedFrequencyTransmon` library now includes an ``angle`` parameter
```
same for the rest of doc.